src/plugin/index.js

```diff
@@ -6,7 +6,9 @@ const importCwd = require("import-cwd"); // used to get the users project detail
 const pkg = importCwd("./package.json");
 
 function Jarvis(options) {
```

Comment:
We can probably use option = {} as a default parameter value then below in the options object no need to check if it's defined. What do you think?